src/templates/fr/2019/accessibility_statement.html

```diff
@@ -0,0 +1,122 @@
+        </p>
+      
+        <h2 id="compatibility"><a href="#compatibility" class="anchor-link">Compatibilité avec les navigateurs et les technologies d’assistance</a></h2>
+
+        <p>
+          Le Web Almanac est conçu pour être compatible avec autant de navigateurs et de technologies d’assistance que possible. Le site Web a été construit de manière progressive et est pleinement fonctionnel même sans prendre en charge toutes les fonctionnalités de navigateur modernes que nous utilisons, mais vous pouvez remarquer une détérioration des visuels et un manque de fonctionnalités (visuels interactifs, nombre de commentaires, etc.) lorsque vous n’utilisez un navigateur moderne tel qu’une version récente de Chrome, Safari, Firefox ou Edge.
+        </p>
+        
+        <p>
+          Pour le moment, nous n’avons pas terminé les tests du matériel ou des logiciels de technologie d’assistance, mais nous avons utilisé des outils, comme mentionné ci-dessus, pour tenter de résoudre les problèmes de ces appareils.
+        </p>
+      </section>
+      
+      <section>
+        <h2 id="technical-specifications"><a href="#technical-specifications" class="anchor-link">Spécifications techniques</a></h2>
+        
+        <p>
+          Le site Web est construit sur du HTML sémantique bien structuré (y compris <a href="https://www.w3.org/WAI/standards-guidelines/aria/" hreflang="en">WAI-ARIA</a> le cas échéant), amélioré avec CSS pour le style et JavaScript limité pour certains composants interactifs. Le site Web utilise des pratiques de développement modernes telles que les polices CSS Grid, Fetch et WOFF2, qui ne sont pas prises en charge dans les navigateurs plus anciens tels que IE 11. Le site Web est toujours entièrement fonctionnel dans ces navigateurs mais peut ne pas être aussi attrayant visuellement. Aucun plug-in ou bibliothèque supplémentaire ne devrait être nécessaire pour afficher le site Web.
+        </p>
+        <p>
+          Le Web Almanac PDF suit les mêmes normes d’accessibilité que le site Web et, en outre, est structurellement étiqueté pour faciliter l’accessibilité.
+        </p>
+      </section>
+      
+      <section>
+        <h2 id="assessment-approach"><a href="#assessment-approach" class="anchor-link">Approche d’évaluation</a></h2>
+        
+        <p>
+          Auto-évaluation&nbsp;: le contenu a été évalué par l’équipe Web Almanac en utilisant sa propre expertise et un certain nombre d’outils dont <a href="https://wave.webaim.org/" hreflang="en">l’outil d’accessibilité WAVE</a>, le <a href="https://hacks.mozilla.org/2019/10/auditing-for-accessibility-problems-with-firefox-developer-tools/" hreflang="en">l’inspecteur d’accessibilité Firefox</a> et le <a href="https://chrome.google.com/webstore/detail/axe-web-accessibility-tes/lhdoppojpmngadmnindnejefpokejbdd" rel="nofollow">plugin AX Chrome</a>. Le Web Almanac PDF a été testé avec une analyse d’accessibilité dans Acrobat Reader Pro DC.
+      </section>
```

Comment:
Missed a `</p>` - bless that linter!:

```suggestion
        </p>
      </section>
```